.travis.yml

```diff
@@ -8,6 +8,7 @@ addons:
   postgresql: "9.3"
 before_install:
   - redis-server --version
+  - conda list
```

Comment:
my problem is, that nosetests pass on my machine, but not on Travis. I figure there are slightly different packages installed :-/
Here is my environment:
_nb_ext_conf              0.3.0                    py27_0    defaults
anaconda-client           1.5.1                    py27_0    defaults
backports                 1.0                      py27_0    defaults
backports-abc             0.4                       <pip>
backports.shutil-get-terminal-size 1.0.0                     <pip>
backports.ssl-match-hostname 3.4.0.2                   <pip>
backports_abc             0.4                      py27_0    defaults
bcrypt                    3.1.0                     <pip>
cffi                      1.7.0                     <pip>
click                     6.6                      py27_0    defaults
clyent                    1.2.2                    py27_0    defaults
configparser              3.5.0                    py27_0    defaults
coverage                  4.2                       <pip>
dbus                      1.10.10                       0    defaults
decorator                 4.0.10                   py27_0    defaults
entrypoints               0.2.2                    py27_0    defaults
enum34                    1.1.6                    py27_0    defaults
expat                     2.1.0                         0    defaults
flake8                    3.0.4                     <pip>
fontconfig                2.11.1                        6    defaults
freetype                  2.5.5                         1    defaults
funcsigs                  1.0.2                     <pip>
functools32               3.2.3.2                  py27_0    defaults
future                    0.13.1                    <pip>
get_terminal_size         1.0.0                    py27_0    defaults
glib                      2.43.0                        1    defaults
gst-plugins-base          1.8.0                         0    defaults
gstreamer                 1.8.0                         0    defaults
icu                       54.1                          0    defaults
ipykernel                 4.5.0                    py27_0    defaults
ipython                   5.1.0                    py27_0    defaults
ipython-genutils          0.1.0                     <pip>
ipython_genutils          0.1.0                    py27_0    defaults
ipywidgets                5.2.2                    py27_0    defaults
jinja2                    2.8                      py27_1    defaults
jpeg                      8d                            2    defaults
jsonschema                2.5.1                    py27_0    defaults
jupyter                   1.0.0                    py27_3    defaults
jupyter-client            4.4.0                     <pip>
jupyter-console           5.0.0                     <pip>
jupyter-core              4.2.0                     <pip>
jupyter_client            4.4.0                    py27_0    defaults
jupyter_console           5.0.0                    py27_0    defaults
jupyter_core              4.2.0                    py27_0    defaults
knight-lab-ui (/media/barnacle/home/sjanssen/AmericanGut/labadmin) 0.1.0.dev0                <pip>
libffi                    3.2.1                         0    defaults
libgcc                    5.2.0                         0    defaults
libpng                    1.6.22                        0    defaults
libpq                     9.5.4                         0    defaults
libsodium                 1.0.10                        0    defaults
libxcb                    1.12                          0    defaults
libxml2                   2.9.2                         0    defaults
libxslt                   1.1.28                        0    defaults
markupsafe                0.23                     py27_2    defaults
mccabe                    0.5.2                     <pip>
mistune                   0.7.3                    py27_0    defaults
mock                      2.0.0                     <pip>
natsort                   5.0.1                     <pip>
nb-anacondacloud          1.2.0                     <pip>
nb-conda                  2.0.0                     <pip>
nb-conda-kernels          2.0.0                     <pip>
nb_anacondacloud          1.2.0                    py27_0    defaults
nb_conda                  2.0.0                    py27_0    defaults
nb_conda_kernels          2.0.0                    py27_0    defaults
nbconvert                 4.2.0                    py27_0    defaults
nbformat                  4.1.0                    py27_0    defaults
nbpresent                 3.0.2                    py27_0    defaults
ncurses                   6.0                           0    rdkit
nose                      1.3.7                     <pip>
notebook                  4.2.3                    py27_0    defaults
numpy                     1.11.1                    <pip>
open-humans-tornado-oauth2 2.1.0                     <pip>
openssl                   1.0.2h                        1    defaults
pandas                    0.18.1                    <pip>
passlib                   1.6.2                     <pip>
path.py                   8.2.1                    py27_0    defaults
pathlib2                  2.1.0                    py27_0    defaults
pbr                       1.10.0                    <pip>
pep8                      1.7.0                     <pip>
pexpect                   4.0.1                    py27_0    defaults
pickleshare               0.7.4                    py27_0    defaults
pillow                    3.3.1                     <pip>
pip                       8.1.2                    py27_0    defaults
prompt-toolkit            1.0.3                     <pip>
prompt_toolkit            1.0.3                    py27_0    defaults
psycopg2                  2.6.2                    py27_0    defaults
ptyprocess                0.5.1                    py27_0    defaults
py-bcrypt                 0.4                       <pip>
pycodestyle               2.0.0                     <pip>
pycparser                 2.14                      <pip>
pycrypto                  2.6.1                     <pip>
pyflakes                  1.2.3                     <pip>
pygments                  2.1.3                    py27_0    defaults
pyqt                      5.6.0                    py27_0    defaults
python                    2.7.13                        0    defaults
python-dateutil           2.5.3                    py27_0    defaults
pytz                      2016.7                   py27_0    defaults
pyyaml                    3.12                     py27_0    defaults
pyzmq                     15.4.0                   py27_0    defaults
qt                        5.6.0                         0    defaults
qtconsole                 4.2.1                    py27_1    defaults
readline                  6.2                           2    defaults
redis                     3.2.0                         0    defaults
requests                  2.11.1                   py27_0    defaults
requests-mock             1.1.0                     <pip>
setuptools                25.1.6                   py27_0    defaults
simplegeneric             0.8.1                    py27_1    defaults
singledispatch            3.4.0.3                  py27_0    defaults
sip                       4.18                     py27_0    defaults
six                       1.10.0                   py27_0    defaults
sqlite                    3.13.0                        0    defaults
ssl_match_hostname        3.4.0.2                  py27_1    defaults
terminado                 0.6                      py27_0    defaults
tk                        8.5.18                        0    defaults
tornado                   4.4.1                    py27_0    defaults
traitlets                 4.3.1                    py27_0    defaults
wcwidth                   0.1.7                    py27_0    defaults
wheel                     0.29.0                   py27_0    defaults
widgetsnbextension        1.2.6                    py27_0    defaults
wtforms                   2.0.1                     <pip>
yaml                      0.1.6                         0    defaults
zeromq                    4.1.4                         0    defaults
zlib                      1.2.8                         3    defaults